NodeOpera/Dockerfile.txt

```diff
@@ -0,0 +1,42 @@
+USER root
+
+#=========
+# Opera
+#=========
+ENV OPERA_VERSION=60.0.3255.170
+ENV OPERA_DRIVER_VERSION=v.2.45
```

Comment:
Do we need to hardcode the driver version?

Reply:
Deleted. I will use link to latest release instead, so hardcoded version is not required.